db/migrate/20211123060535_create_treatment_inertia_tables.rb

```diff
@@ -0,0 +1,35 @@
+class CreateTreatmentInertiaTables < ActiveRecord::Migration[5.2]
+  def up
+    create_table :clean_medicine_to_dosage, id: false do |t|
+      t.bigint :rxcui, null: false
+      t.string :medicine, null: false
+      t.float :dosage, null: false
+    end
+    execute "COPY clean_medicine_to_dosage FROM '#{Rails.root}/config/data/treatment-inertia/clean_medicine_to_dosage.csv' WITH DELIMITER ',' CSV HEADER"
+
+    create_table :raw_to_clean_medicine, id: false do |t|
+      t.string :raw_name, null: false
+      t.string :raw_dosage, null: false
+      t.bigint :rxcui, null: false
+    end
+    execute "COPY raw_to_clean_medicine FROM '#{Rails.root}/config/data/treatment-inertia/raw_to_clean_medicine.csv' WITH DELIMITER ',' CSV HEADER"
+
+    create_table :medicine_purpose, id: false do |t|
+      t.string :name, null: false
+      t.boolean :hypertension, null: false
+      t.boolean :diabetes, null: false
+    end
+    execute "COPY medicine_purpose FROM '#{Rails.root}/config/data/treatment-inertia/medicine_purpose.csv' WITH DELIMITER ',' CSV HEADER"
```

Comment:
The `COPY`s can be split into a data migration.

Reply:
Done in ebc10d430924d013f63a21b9e15531017cbe4627.